### ISSUE TITLE
wrap (rather than truncate) long lines in listsTable

### ISSUE THREAD

**Issue by hawkeye116477** [collinbarrett/FilterLists#494]
## Bug
Just like in title. My proposition for fix is to add `.ReactTable .rt-tbody .rt-td{white-space: normal;}` to css.

Before
![screenshot_20180917_225623](https://user-images.githubusercontent.com/19818572/45649934-0df4a900-bacd-11e8-9e66-ece6c26bc13d.png)

After
![screenshot_20180917_225827](https://user-images.githubusercontent.com/19818572/45649997-367ca300-bacd-11e8-993f-01b4db5af73a.png)

**Comment by collinbarrett** [comment]
I like the idea. I think, as it stands, if the line wraps to two lines, it makes the height of the row inconsistent with one-line titles. So, clicking "Next"/"Previous" at the bottom becomes more laborious.

The immediate workaround is to hide one of the columns to the right with the checkboxes at the bottom if you need to see a full title.

I am preparing a refactor that would surface a lot more data in the main table. Such as #455 and #488 .

Once that is complete, we'll circle back around to this.

**Comment by collinbarrett** [comment]
Closing as won't fix for now. The alternative is to hide more columns using the checkboxes below the table so that you can see more of the title. Wrapping the lines creates inconsistent row heights which makes toggling through the pages inconvenient.

Open to reconsider, but I think tweaking the columns that are visible is a reasonable solution for now.
